databroker/src/grpc/kuksa_val_v2/conversions.rs

```diff
@@ -267,8 +267,8 @@ fn value_restriction_from(metadata: &broker::Metadata) -> Option<proto::ValueRes
                     r#type: Some(proto::value_restriction::Type::Signed(
                         proto::ValueRestrictionInt {
                             allowed_values: allowed,
-                            min: min_value,
-                            max: max_value,
+                            min: min_value.unwrap_or(i64::MIN),
```

Comment:
Will this not result in odd results for smaller VSS-types like `int8` where you for instance have `max: 53` but not any minimum defined. Will we then not return `i64::MIN` which is not correct and might be confusing for the client? 

I.e. do we need some helper method `get_implicit_min(metadata.data_type)` that returns for instance -128 for a `int8`

Reply:
This was removed and `None` placed instead, so it should be resolved